app/decorators/staff/time_slots_decorator.rb

```diff
@@ -1,4 +1,4 @@
-class TimeSlotsDecorator < Draper::CollectionDecorator
+class Staff::TimeSlotsDecorator < Draper::CollectionDecorator
```

Comment:
Sure, TimeSlots along with other models such as ProgramSessions, Rooms, Teammates are staff only concerns.  Since we don't necessarily scope all models this way it feels inconsistent.  The only model that I know has different use in staff vs non-staff is proposals.  Even then it's got a three way concern of speaker vs reviewer vs program.  My observation isn't a biggie either way but since you purposefully moved the file, it caught my eye.